docs/developers/celestia-node-metrics.md

```diff
@@ -19,7 +19,7 @@ command:
 
 <!-- markdownlint-disable MD013 -->
 ```sh
-celestia light start --core.ip <ip-address> --core.grpc.port <port> --metrics --metrics.endpoint <ip-address:port>
+celestia light start --core.ip <ip-address> --core.grpc.port <port> --metrics --metrics.endpoint <ip-address:port> --gateway --gateway.ip <ip-address> --gatway.port <port>
```

Comment:
please edit to `--gateway.addr` instead of `--gateway.ip` 🙏

Reply:
yes - adding to #317